client/packages/programs/src/JsonForms/components/ProgramSearch.tsx

```diff
@@ -31,16 +31,26 @@ const UIComponent = (props: ControlProps) => {
 
   const { handleChange, label, path } = props;
   const { core } = useJsonForms();
-  const { data, isLoading } = useProgramList({
-    isImmunisation: props.uischema.options?.['programType'] === 'immunisation',
-  });
+
+  const { data, isLoading } = useProgramList(
+    props.uischema.options?.['programType'] === 'immunisation'
+      ? {
+          isImmunisation: true,
+        }
+      : {}
+  );
```

Comment:
Make Program input generic so not just for immunisation / not immunisation searches.